app/assets/stylesheets/common/bootstrap_custom.scss

```diff
@@ -0,0 +1,50 @@
+// Core variables and mixins
+@import "bootstrap/variables";
+@import "bootstrap/mixins";
+
+// Reset and dependencies
+@import "bootstrap/normalize";
+@import "bootstrap/print";
+@import "bootstrap/glyphicons";
+
+// Core CSS
+@import "bootstrap/scaffolding";
+@import "bootstrap/type";
+@import "bootstrap/code";
+@import "bootstrap/grid";
+@import "bootstrap/tables";
+@import "bootstrap/forms";
+@import "bootstrap/buttons";
+
+// Components
+@import "bootstrap/component-animations";
+@import "bootstrap/dropdowns";
+@import "bootstrap/button-groups";
+@import "bootstrap/input-groups";
+@import "bootstrap/navs";
+@import "bootstrap/navbar";
+@import "bootstrap/breadcrumbs";
+@import "bootstrap/pagination";
+@import "bootstrap/pager";
+@import "bootstrap/labels";
+@import "bootstrap/badges";
+@import "bootstrap/jumbotron";
+@import "bootstrap/thumbnails";
+@import "bootstrap/alerts";
+@import "bootstrap/progress-bars";
+@import "bootstrap/media";
+@import "bootstrap/list-group";
+@import "bootstrap/panels";
+@import "bootstrap/responsive-embed";
+@import "bootstrap/wells";
+@import "bootstrap/close";
+
+// Components w/ JavaScript
+@import "bootstrap/modals";
+@import "bootstrap/tooltip";
+@import "bootstrap/popovers";
+@import "bootstrap/carousel";
+
+// Utility classes
+@import "bootstrap/utilities";
+@import "bootstrap/responsive-utilities";
```

Comment:
I think this is good. We can easily comment out the components we r not
using.

Sent from iPhone


On Mar 6, 2015, at 5:12 PM, Vipul A M <notifications@github.com> wrote:

In app/assets/stylesheets/common/bootstrap_custom.scss
<https://github.com/bigbinary/wheel/pull/60#discussion_r25982715>:

> +@import "bootstrap/media";
> +@import "bootstrap/list-group";
> +@import "bootstrap/panels";
> +@import "bootstrap/responsive-embed";
> +@import "bootstrap/wells";
> +@import "bootstrap/close";
> +
> +// Components w/ JavaScript
> +@import "bootstrap/modals";
> +@import "bootstrap/tooltip";
> +@import "bootstrap/popovers";
> +@import "bootstrap/carousel";
> +
> +// Utility classes
> +@import "bootstrap/utilities";
> +@import "bootstrap/responsive-utilities";

can't we so single import here?

—
Reply to this email directly or view it on GitHub
<https://github.com/bigbinary/wheel/pull/60/files#r25982715>.